iml-manager-cli/src/server.rs

```diff
@@ -373,6 +391,135 @@ pub async fn server_cli(command: ServerCommand) -> Result<(), ImlManagerCliError
 
             wait_for_cmds(cmds).await?;
         }
+        ServerCommand::Remove(config) => {
+            let term = Term::stdout();
+
+            let remove_hosts = hostlist_parser::parse(&config.hosts)?;
+
+            tracing::debug!("Parsed hosts {:?}", remove_hosts);
+
+            let (_, api_hosts) = future::try_join(get_hosts(), get_hosts()).await?;
+
+            let known_hosts = filter_known_hosts(&remove_hosts, &api_hosts.objects);
+
+            let objects = known_hosts
+                .iter()
+                .map(|host| TestHostConfig {
+                    address: host.address.as_str(),
+                    auth_type: "existing_keys_choice".into(),
+                })
+                .collect();
+
+            let Objects { objects } = post("test_host", Objects { objects })
+                .await?
+                .error_for_status()?
+                .json()
+                .await
+                .map_err(iml_manager_client::ImlManagerClientError::Reqwest)?;
+
+            let cmds = objects
+                .into_iter()
+                .map(|CmdWrapper { command }| command)
+                .collect();
+
+            term.write_line(&format!("{} preflight checks...", style("Running").green()))?;
```

Comment:
Are any checks really needed before removing the servers?

Reply:
IMO not needed, but in original issue it marked as needed.